src/main/webapp/WEB-INF/view/chat.jsp

```diff
@@ -75,20 +75,53 @@ String membersOfConversation = (String) request.getAttribute("membersOfConversat
       e.preventDefault();
       if(!this.classList.contains('isDisabled')){
 
+        var userName = this.getAttribute("username");
+
         this.removeAttribute("href");
         this.classList.add('isDisabled');
         var selectedUserList = document.getElementById('selectedUserList');
         var selectedUser = document.createElement("button");
         selectedUser.setAttribute("class", "btn btn-primary");
         selectedUser.classList.add('selectedUser');
-        selectedUser.innerHTML = this.getAttribute("username");
+        selectedUser.innerHTML = userName;
+
+        var closeIcon = document.createElement("button");
+        closeIcon.setAttribute("type", "button");
+        closeIcon.setAttribute("class", "close");
+        closeIcon.setAttribute("aria-label", "Close");
+        closeIcon.setAttribute("id", "remove-from-queue-button");
```

Comment:
We shouldn't be setting the ID of more than one element to the same value. A couple of alternative solutions here: Either add a click listener to this button before adding it:
`closeIcon.addEventListener('click', () => $(selectedUser).remove();`
Or use the class to listen on element clicks, like you did on line 74, and add a custom attribute `username`, also like you did for the add user button.
The first solution is both easier and cleaner.

Reply:
Thanks, will fix it now.